### ISSUE TITLE
Liquibase CLI "validation" command returns success status on a validation errors

### ISSUE THREAD

**Issue by vpinyagin** [liquibase/liquibase#1630]
## Environment
**Liquibase Version**: 3.8.0, 4.1.1

**Liquibase Integration & Version**: CLI

**Database Vendor & Version**: Postgress

**Operating System Type & Version**: Linux

## Description
The Liquibase CLI has the "validate" command to check is a changelog valid.
When this command is run on an invalid changelog.xml it finds and reports an error (that is correct) and after then it reports 
"_Liquibase command 'validate' was executed successfully._" and returns zero (success) exit status that is completely discouraged and allow a deployment scripts to execute corrupted changelogs as long as the validation step has been "successfully" passed. 

## Steps To Reproduce
Make a changeSet with duplicated id:

```
    <changeSet author="vpinyagin" id="A013" context="ctx1">
        <tagDatabase tag="A013"/>
    </changeSet>

    <changeSet author="vpinyagin" id="A013" context="ctx1">
        <tagDatabase tag="A014"/>
    </changeSet>

```

Run "liquibase validate".

Check exit code, it must be non-zero, but actually it is zero (success).

## Actual Behavior
Reports  validation **error** then reports validation **succeed** and return **zero** exit status.

## Expected/Desired Behavior
Reports validation **error** then reports validation **failed** and return **non-zero** exit status.

## Screenshots (if appropriate)
![Screenshot!UNITO-UNDERSCORE!2021-01-15!UNITO-UNDERSCORE!16-05-45](https://user-images.githubusercontent.com/2586920/104717926-e9b44000-574b-11eb-8733-4c15d306deeb.jpeg)

**Comment by molivasdat** [comment]
Hi @vpinyagin Thanks for bringing this issue to our attention.  I believe this would be a change in how the command works today.  Thanks for the excellent use case on how this should work.  We will add this to the list of issues we are processing.

**Comment by FBurguer** [comment]
Its actually fixed for liquibase `4.14`, my mistake.  Im closing this issue, if it continues to happen to you please re-open the issue. Thanks
